clap-tests/run_tests.py

```diff
@@ -213,6 +229,8 @@
 		'mult_valsmo x2-1:   ': ['{} --multvalsmo some other --multvalsmo some'.format(_bin), _mult_vals_2m1],
 		'mult_valsmo x1:     ': ['{} --multvalsmo some other'.format(_bin), _exact],
 		'F2(ss),O(s),P:      ': ['{} value -f -f -o some'.format(_bin), _f2op],
+        'arg dym:            ': ['{} --optio=foo'.format(_bin), _arg_dym_usage],
+        'pv dym:             ': ['{} --Option slo'.format(_bin), _pv_dym_usage],
```

Comment:
Oops, here I see a white-space error. My editor used spaces for a while, which is standard for python, whereas the file used tabs. This went unnoticed as it's apparently not an error to do that while defining a dict.
This might want to be adjusted at some point, maybe you can give it a quick-fix-commit.